FlightsWebsite/src/js/flights.js

```diff
@@ -26,4 +124,174 @@ function status(response) {
 
 function json(response) {
     return response.json()
-}
\ No newline at end of file
+}
+
+function showTotalFlights(data) {
+
+    let chart = new CanvasJS.Chart("chartContainer", {
+        animationEnabled: true,
+        theme: "light2", // "light1", "light2", "dark1", "dark2"
+        title: {
+            text: "Total number of flights per month"
+        },
+        axisY: {
+            title: "Total number of flights",
+            interval: 5000
+        },
+        axisX: {
+            interval: 1
+        },
+        data: [{
+            type: "column",
+            showInLegend: true,
+            legendMarkerColor: "grey",
+            legendText: "Month",
+            dataPoints: data
+        }]
+    });
+    chart.render();
+}
+
+function showTopDestinations(data) {
```

Comment:
Are you sure that it should not be divided by destinations? The requirement says: 
"The top-10 destinations and how many flights were made to these
-For these 10 destinations, make a visualization of the number of flights from the three origins to the top-10 destination."
For me it a bit sounds like it should be divided by those 3 destinations, but I'm not sure. Maybe we should ask teacher what they mean?

Reply:
I agree with Michal .. I would firstly as you did find Top-10 .. and then for each of these Top-10 find how many were at which origin .. the question what I would ask teachers is that if we have to provide two graphs in this case - one for "The top-10 destinations and how many flights were made to these" and second for "For these 10 destinations, make a visualization of the number of flights from the three origins to the top-10 destination" .. or only one in this case for "For these 10 destinations, make a visualization of the number of flights from the three origins to the top-10 destination"